### PR TITLE
WF-65219-Correct namespace in Presentation UG

### PR DESCRIPTION
**Pull request by sharonaumao**
<table style="width: 521.306px;">
<tbody>
<tr>
<td style="width: 156px;">Task Link</td>
<td style="width: 349.306px;">https://syncfusion.atlassian.net/browse/WF-65219</td>
</tr>
<tr>
<td style="width: 156px;">Description</td>
<td style="width: 349.306px;">Correcting namespace in Presentation UG</td>
</tr>
<tr>
<td style="width: 156px;">Solution</td>
<td style="width: 349.306px;">Added namespaces to the following md files-Getting-Started, word-to-pdf</td>
</tr>
<tr>
<td style="width: 156px;">Content review confirmation</td>
<td style="width: 349.306px;">Yes</td>
</tr>
</tbody>
</table>

[WF-65091]: https://syncfusion.atlassian.net/browse/WF-65091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]
Add namespace for core and xamarin for PPTX to PDF code snippet in getting started page
https://help.syncfusion.com/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-pdf

File: File-Formats/Presentation/Getting-Started.md [hunk 1]
```diff
@@ -549,6 +549,20 @@ Imports Syncfusion.Pdf
 Imports Syncfusion.PresentationToPdfConverter
 {% endhighlight %}
 
+{% highlight c# tabtitle="ASP.NET Core" %}
+using Syncfusion.Presentation;
+using Syncfusion.OfficeChartToImageConverter;
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Getting-Started.md:554]: Remove OfficeChartToImageConverter and add System.IO

The namespace is similar to PPTX to PDF page.

File: File-Formats/Presentation/Getting-Started.md [hunk 2]
```diff
@@ -549,6 +549,20 @@ Imports Syncfusion.Pdf
...
+using Syncfusion.Pdf;
+using Syncfusion.PresentationRenderer;
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+using Syncfusion.Presentation;
+using Syncfusion.OfficeChartToImageConverter;
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Getting-Started.md:561]: Remove OfficeChartToImageConverter and add System.IO

The namespace is similar to PPTX to PDF page.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/Getting-Started.md (hunk 1 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Getting-Started.md]: @Suriya-Balamurugan -Modified

File: File-Formats/Presentation/Getting-Started.md (hunk 2 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Getting-Started.md]: @Suriya-Balamurugan -Modified

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**